### PR TITLE
LIT 5985-86 6367-68

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New records, further work related to teh Amharic commentary on Psalter

### CODE REVIEW

**Review by Ralph-Lee-UK — APPROVED** [review]

File: 5001-6000/LIT5986CommPsalter.xml [hunk 1]
```diff
@@ -41,20 +47,69 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
             <div type="bibliography">
             <listRelation>
                 <relation name="saws:isCommentOn" active="LIT5986CommPsalter" passive="LIT2701Dawit"/>
-            </listRelation>
+                <relation name="lawd:hasAttestation" active="LIT5986CommPsalter" passive="ESrqg015"/>
+                <relation name="lawd:hasAttestation" active="LIT5986CommPsalter" passive="BLorient535"/>
+    </listRelation>
+                <div type="edition">
+                  <note>The division into text parts follows <ref type="mss" corresp="BLorient353"/>.</note>
+                   <div type="textpart" subtype="incipit" xml:id="LIT5985CommPsalms">
```
Comment (thea-m) [inline 5001-6000/LIT5986CommPsalter.xml:68]: please add here `xml:lang="am"` (in the upper div="edition" or in each textpart)

File: new/LIT6368PsaltComm.xml [hunk 2]
```diff
@@ -0,0 +1,121 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6368PsaltComm" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Amharic Commentary on the Psalter</title>
+                  <title xml:lang="gez" xml:id="t2">ትርጓሜ፡ ዳዊት፡</title>
+                  <title xml:lang="gez" corresp="#t2" type="normalized">Tərgwāme Dāwit</title>
```
Comment (thea-m) [inline new/LIT6368PsaltComm.xml:10]: can you write w in superscript?

File: new/LIT6368PsaltComm.xml [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+                    <relation name="saws:contains" active="LIT6368PsaltComm" passive="LIT5986CommPsalter"/>
+                    <relation name="lawd:hasAttestation" active="LIT6368PsaltComm" passive="BNFabb41"/>
+                </listRelation>
+
+           <div type="edition">
+             <note>The division into text parts follows <ref type="mss" corresp="BNFabb41"/>.</note>
+              <div type="textpart" subtype="incipit" xml:id="Psalms">
+                 <label>Psalms of David with Amharic commentary</label>
+                 <ab>
+                   <l n="1"/>ተግሣጽ፡ ለጻድቃን፡ ወለኃጥአን።
```
Comment (thea-m) [inline new/LIT6368PsaltComm.xml:73]: I cannot open the guidelines now (they don't load), but I think that the verse should be contained in `<l></l>` (as you have done above).

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you!

File: new/LIT6368PsaltComm.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply new/LIT6368PsaltComm.xml:73]: ok

**Review by thea-m — APPROVED** [review]

File: 5001-6000/LIT5986CommPsalter.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply 5001-6000/LIT5986CommPsalter.xml:68]: done

**Pull request merged**